cmd/create/accountroles/cmd.go

```diff
@@ -398,7 +414,7 @@ func createRoles(r *rosa.Runtime, prefix, permissionsBoundary, accountID, env st
 		}
 
 		filename := fmt.Sprintf("sts_%s_trust_policy", file)
-		policyDetail := aws.GetSTSPolicyDetails(policies, filename)
+		policyDetail := aws.GetManagedPolicyDetails(policies, filename)
```

Comment:
Always get the managed policy? Even if the bool is false?

Reply:
Good catch, lousy naming, updated to `GetPolicyDetails`.
`GetPolicyDetails` applies both for unmanaged and managed policies, the function was added for this change https://github.com/openshift/rosa/pull/986/files#diff-97477d2275d7797e234adfc6164c02beabfdbdb0eb516faed787f52d1cfb30c1R521.

The other function `GetManagedPolicyARN` applies only for managed policies.